### PR TITLE
Python compiler print true

### PR DESCRIPTION
**Pull request by prvshah51**
Made some basic changes in the files to make Python compiler work.

### CODE REVIEW

**Comment by MikaelMayer** [comment]
Great !
Could you rather base your pull request against master? It seems that you are basing it against an old version of your branch and your branch has some commits since master, which would make it difficult to review. thanks.

File: Source/Dafny/ConcreteSyntax/ConcreteSyntaxTreeUtils.cs [hunk 1]
```diff
@@ -55,15 +55,29 @@ public static class ConcreteSyntaxTreeUtils {
...
+          outer.WriteLine();
+          PythonCompiler.indent += 1;
           break;
       }
 
-      outer.WriteLine("{");
-      body = outer.Fork(1);
-      outer.Write("}");
+
+      body = outer.Fork((open == BraceStyle.Pindent) ? PythonCompiler.indent : 1);
```
Comment (keyboardDrummer) [inline Source/Dafny/ConcreteSyntax/ConcreteSyntaxTreeUtils.cs:71]: I'm confused, why doesn't 1 work? The `1` is relative to the existing indentation right?

Comment (prvshah51) [reply Source/Dafny/ConcreteSyntax/ConcreteSyntaxTreeUtils.cs:71]: At first I didn't know about the _relativeIndentLevel_ that's why initiated _PythonCompiler.indent_ and was using that, despite the name says it's relative I assumed it being global and added _PythonCompiler.indent_ in the equation there. Thank for the input I will make changes to that.

File: README.md [hunk 2]
```diff
@@ -19,13 +19,9 @@ You can ask questions about Dafny on [Stack Overflow](https://stackoverflow.com/
...
 # Try Dafny
 
-The easiest way to get started with Dafny is to use [rise4fun](http://rise4fun.com/dafny), where you can write and verify Dafny programs without having install anything. On [rise4fun](http://rise4fun.com/dafny), you will also find the online Dafny tutorial.
-It is also easy to [install Dafny on your own machine](https://github.com/dafny-lang/dafny/wiki/INSTALL) in VS Code, which gives you a much better user experience than in the web browser.
-
-# Setup
-
-See [installation instructions](https://github.com/dafny-lang/dafny/wiki/INSTALL) on the wiki
-and instructions for installing the [Dafny mode for Emacs](https://github.com/boogie-org/boogie-friends).
+The easiest way to try out Dafny is to [install Dafny on your own machine in Visual Studio Code](https://github.com/dafny-lang/dafny/wiki/INSTALL#visual-studio-code)
```
Comment (keyboardDrummer) [inline README.md:22]: Are these your changes? Might be nice to have them in a separate PR

**Pull request closed without merging**